samu.c

```diff
@@ -90,6 +91,50 @@ jobsflag(const char *flag)
 	buildopts.maxjobs = num > 0 ? num : -1;
 }
 
+static void
+jobserverflags(const char *flag)
+{
+	int rfd, wfd;
+
+	if (!flag)
+		return;
+	if (sscanf(flag, "%d,%d", &rfd, &wfd) == 2) {
+		/* prepare error message */
+		errno = EBADF;
```

Comment:
This is a success case, right?

Reply:
Yes. Since it's no longer validating the FDs with POLLNVAL below, setting errno here is just obsolete.